### PR TITLE
Moved and renamed files to more standardize naming

### PR DESCRIPTION
**Pull request by Liriaene**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: api/accounts/getAccount.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import * as auth from './authenticate';
```
Comment (gstone271) [inline api/accounts/getAccount.ts:1]: I don't like this naming scheme because it has the word `account` multiple times, redundantly.

**Review by OkapalDominic — COMMENTED** [review]
You can create an index.ts file in each of the folders which exports each endpoint, for example in accounts/index.ts
export { default as changePassword } from './changePassword';
export { default as changeUsername } from './changeUsername';
export { default as create } from './createAccount';
export { default as get } from './getAccount';
export { default as reset } from './resetPassword';

Then in app.ts you only need one import:
import * as accounts from './accounts';

And you can then remove all the scattered imports so instead of:
import accountsGetEndpoint from './accounts/getAccount';
app.get('/accounts', accountsGetEndpoint);

just do:
app.get('/accounts', accounts.get);

**Comment by ElijahRW** [comment]
Please rebase branch

**Comment by Liriaene** [comment]
This PR has been moved to #97

**Pull request closed without merging**